btrdb/utils/general.py

```diff
@@ -36,7 +35,7 @@ def unpack_stream_descriptor(desc):
     return tags, anns
 
 
-def batched(iterable: Iterable[Any], n: int) -> Generator[Any, Any, Any]:
```

Comment:
I think the Iterable would have been fine to use here if it was imported from `typing.Iterable`
but the Iterable [here](https://docs.python.org/3.8/library/typing.html#:~:text=Iterable%5BLoggedVar%5Bint%5D%5D)

Reply:
i think the previous fail for walrus operator might also be from the typing hint was from using the collection's `Iterable`.

Reply:
It was imported from typing `Iterable` before. I think the `Generator` isn't supported in earlier versions of Python and the variable is already named `iterable` so the partial type hint doesn't do much.